.ci_support/osx_64_r_base4.1.yaml

```diff
@@ -4,12 +4,24 @@ c_compiler:
 - clang
 c_compiler_version:
 - '14'
+cairo:
+- '1'
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 cran_mirror:
 - https://cran.r-project.org
+freetype:
+- '2'
+harfbuzz:
+- '6'
+icu:
+- '70'
+libjpeg_turbo:
+- 2.1.5
```

Comment:
I'm confused why this is here (R4.1) and yet the recipe still works. But I've checked the artifacts and everything looks good so... 🤷

Reply:
yeah, the rendering will try to  specify everything it sees in the recipe. not too smart.